src/components/Inputs/Select/Select.tsx

```diff
@@ -8,45 +8,53 @@ import {
   Select as SelectDS,
   SelectChangeEvent,
 } from "@tracktor/design-system";
-import { forwardRef, Ref, useCallback, useState } from "react";
+import { forwardRef, Ref, useEffect, useState } from "react";
 import useInputs from "@/hooks/useInputs";
 import ChangeEventField from "@/types/ChangeEventField";
 import type TreeNode from "@/types/TreeNode";
 
-export interface SelectProps<T = unknown> {
+export interface SelectProps {
   data: TreeNode;
   helperText?: string;
   inputRef: Ref<unknown>;
   required?: boolean;
-  defaultValue?: T;
   readOnly?: boolean;
-  onChange?(dataAttribute: ChangeEventField): void;
+  handleFormValue?(dataAttribute: ChangeEventField): void;
+  value?: string;
 }
 
-const Select = ({ data, helperText, inputRef, required, onChange, readOnly, defaultValue = "" }: SelectProps, ref: Ref<HTMLDivElement>) => {
+const Select = ({ data, helperText, inputRef, required, handleFormValue, readOnly, value }: SelectProps, ref: Ref<HTMLDivElement>) => {
```

Comment:
Ici tu peux initialiser à string vide directement dans la props en param. D'ailleurs tu peux le faire à tous les champs je pense pour éviter d'avoir l'erreur react controller/uncontrolled

```suggestion
const Select = ({ data, helperText, inputRef, required, handleFormValue, readOnly, value = "" }: SelectProps, ref: Ref<HTMLDivElement>) => {
```